docs/releasing.md

```
@@ -4,24 +4,39 @@
 manage releases.
 
 Note: after completing the release, commits must be manually pushed
-from `master` to `production` branch.
+from `main` to `production` branch.
 
 Refer to https://www.vortextemplate.com/docs/workflows/releasing for more information.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)